app/model/des/CustomerInformation.scala

```diff
@@ -35,15 +37,34 @@ object CustomerInformation {
   implicit val format: OFormat[CustomerInformation] = Json.format[CustomerInformation]
 }
 
-final case class ApprovedInformation(bankDetails: Option[BankDetails]) {
+final case class ApprovedInformation(bankDetails: Option[BankDetails], PPOB: Option[PPOB]) {
 
   val bankDetailsExist = bankDetails.isDefined
+
+  def getAddress(vrn: Vrn): Address = PPOB match {
```

Comment:
val maybeAddress = for {
p <- ppob
a <- p.address
} yield a 
val address = maybeAddress.getOrElse(throes ...)